scripts/lib/api/mergeClassMembers.ts

```diff
@@ -127,21 +127,18 @@ async function parseMarkdownIncreasingHeading(
   md: string,
   depthIncrease: number,
 ): Promise<Root> {
-  const root = await unified()
```

Comment:
I removed the `await` because vs code says: `'await' has no effect on the type of this expression`.